view-and-share/README.md

```diff
@@ -0,0 +1,44 @@
+## Pryv view and share data example 
+
+Web app for data visualization & sharing with third parties.
+
+## Story
+
+In the previous tutorial [Collect Survey Data](https://github.com/pryv/app-web-examples/tree/master/collect-survey-data), you have collected and stored data from the mother in her Pryv.io account when she answered the example survey.
+
+In this web app, you provide her with a tool to visualize data from her account and to share it with third parties.
+
+| Sign in                                                 | Visualize data                                                  | Share data                                                      |
+| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| <img src="images/1-login.png" alt="base" style="zoom:33%;" /> | <img src="images/2-visualize.png" alt="consent" style="zoom:33%;" /> | <img src="images/3-sharing.png" alt="form" style="zoom:33%;" /> |
+
+## Project Specifications
+
+- Ask for login
+- Request access for the app "pryv-example-view-and-share" to read the streams "Baby" and "Body"
+- Display the data
+- Create or delete a sharing (URL or email) to a third party
```

Comment:
URL or email??

Reply:
I was specifying the way to share data (clicking on a link or sending the link by email) but it's not clear, so I will only keep "Create or delete a sharing to a third party"